pom.xml

```diff
@@ -92,9 +92,9 @@
         <commons.lang.version>3.13.0</commons.lang.version>
         <junit.jupiter.version>RELEASE</junit.jupiter.version>
         <minio.version>8.2.2</minio.version>
-        <slf4j.version>1.7.32</slf4j.version>
-        <springboot.dependencies.version>2.6.6</springboot.dependencies.version>
-        <springcloud.dependencies.version>2021.0.1</springcloud.dependencies.version>
+        <slf4j.version>2.0.12</slf4j.version>
+        <springboot.dependencies.version>3.2.3</springboot.dependencies.version>
+        <springcloud.dependencies.version>2023.0.0</springcloud.dependencies.version>
         <springfox.version>2.9.2</springfox.version>
```

Comment:
Is this dependency used anywhere ? Because springfox is not compatible with spring boot 3, if it is used you should switch to springdoc openapi otherwise I think it should be deleted

Reply:
I deleted it since it was only used by CSA, it will be written in the corresponding release note